packages/editor/src/client/routes/articleEditor.tsx

```diff
@@ -221,12 +220,10 @@ export function ArticleEditor({id}: ArticleEditorProps) {
     }
   }
 
-  async function handlePublish(publishDate: Date, updateDate: Date, noSlug: boolean) {
+  async function handlePublish(publishDate: Date, updateDate: Date) {
+    const noSlug = metadata.slug === ''
```

Comment:
i think the best way to check here is `!metadata.slug` and you can do that directly in the `if (...)`